File: c/main/hello-world.c
#include "lib/hello-time.h"
#include "main/hello-greet.h"
#include <stdio.h>

int main(int argc, char* argv[]) {
  char array[256];
  if (argc > 1) {
	get_greet(array, argv[1]);
    	
  } else {
  	get_greet(array, "CTW");
  }
  printf("%s\n", array);
  print_localtime();
  return 0;
}

File: python/main/hello-world.py
from lib.hellotime import print_localtime
from hellogreet import get_greet
import sys

if __name__ == "__main__":
	who = "CTW"
	if len(sys.argv) > 1:
		who = (sys.argv)[1]
	print(get_greet(who))
	print_localtime()
